queries/sql/highlights.scm

```diff
@@ -199,6 +238,8 @@
   (keyword_and)
   (keyword_or)
   (keyword_not)
+  (keyword_by)
+  (keyword_on)
 ] @keyword.operator
```

Comment:
Are these all casts (those with primitive-like names)? Like turn a float to int with int(2.5)?

Reply:
I am not sure if I understand that question.
`keyword_on` is mostly used in join statements
`select a.col1, b.col1 from a left join b on a.id=b.id`

`keyword_by` is used in conjunction with `keyword_order`, `keyword_group` or `keyword_partition`. But in some dialects also with `partitioned` or `sort`...